front-end/src/App.css

```diff
@@ -0,0 +1,55 @@
+body{
+    background: rgb(239, 255, 217);
+}
+.post{
+    /* display: inline-grid;
+    grid-template-columns: auto auto auto ;
+    gap: 10px;
+    justify-content: center; */ 
+
+    /* border: 2px solid beige;
+    box-shadow: 2px 3px black;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap; */
+
+    /* justify-content: space-around; */
+
+    /* gap: 20px;
+    overflow-y: unset; */
+    
+}
+::-webkit-scrollbar{
+    width: 10px;
+}
+::-webkit-scrollbar-track{
+    background: #f1f1f1 ;
+}
+::-webkit-scrollbar-thumb{
+    background: rgb(245, 225, 159);
+}
+::-webkit-scrollbar-thumb:hover{
+    background: rgb(255, 232, 188);
+}
+
+.activity{
+    /* display: flex;
+    gap: 20px;
+    align-self:center; */
+}
+
+.userPic{
+}
+.userInfo{
+    
+}
+.descreptionBox{
+    /* align-self: center; */
+    
+}
+.book{
+    /* display: flex;
+    flex-direction: column;
+    align-self: flex-start; */
+
+}
```

Comment:
remove `.userPic`, `.userInfo`, `.descreptionBox` and .`book` since some are empty and the rest have commented out code,
Make sure you add one empty line at the end of the file